### PR TITLE
Gds 397/add commercial type data to cxense

### PR DESCRIPTION
**Pull request by Xayer**
That way we can show a danish "annonce" label, while being able to sort by advertorial in the Cxense backend!

### CODE REVIEW

**Review by MFlor — DISMISSED** [review]

**Comment by sulfo** [comment]
Remember to update the version :)

File: src/Bonnier/WP/Cxense/Assets/Scripts.php [hunk 1]
```diff
@@ -79,6 +79,11 @@ public function head_tags()
                 $recs_tags[$this->org_prefix . 'taxo-tag'] = $this->objects_to_array(get_the_tags());
             }
 
+            if ($commercialType = get_field('commercial_type', $post->ID)) {
```
Comment (alfhen) [inline src/Bonnier/WP/Cxense/Assets/Scripts.php:82]: This logic couples the wp-cxense plugin to closely together with the contenthub-editor plugin

It should instead be added to the gds-frontend project by using the filter: 
```
add_filter('cxense_head_tags', function($recs_tags){
 // do your magic here
});
```

**Review by alfhen — CHANGES_REQUESTED** [review]
Please change

**Comment by Xayer** [comment]
@alfhen alright, I've added the filter here [gds-frontend#822](https://github.com/BenjaminMedia/gds-frontend/pull/822)

**Pull request closed without merging**